pom.xml

```diff
@@ -263,7 +264,7 @@
 
     <properties>
         <jdk.version>6</jdk.version>
-        <jdk>${env.JAVA_HOME_6}</jdk>
+        <jdk>${env.JAVA_HOME_8}</jdk>
```

Comment:
Why is this change needed for the Maven wrapper?

Reply:
Because threre isn' t required to install jdk 6 - tests can be run on jdk 8.